src/overlays/actors/ovl_En_Zod/z_en_zod.c

```diff
@@ -28,62 +60,592 @@ const ActorInit En_Zod_InitVars = {
+        if (this->headRot.x > 0xBB8) {
+            this->headRot.x = 0xBB8;
+        } else if (this->headRot.x < -0xBB8) {
+            this->headRot.x = -0xBB8;
+        }
+    } else {
+        Math_SmoothStepToS(&this->headRot.x, 0, 6, 0x1838, 0x64);
+        Math_SmoothStepToS(&this->headRot.y, 0, 6, 0x1838, 0x64);
+    }
+}
+
+s32 EnZod_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx) {
+    EnZod* this = THIS;
+
+    if (limbIndex == TIJO_LIMB_HEAD) {
+        rot->x += this->headRot.y;
+        rot->z += this->headRot.x;
+    }
+
+    return false;
+}
+
+void EnZod_PostLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx) {
+    if (limbIndex == TIJO_LIMB_HEAD) {
+        Matrix_MultVec3f(&D_80BB0580, &thisx->focus.pos);
+    }
+}
+
+void EnZod_DrawDrums(EnZod* this, PlayState* play) {
+    s32 i;
+    static Gfx* sTijoDrumsDLs[] = {
+        gTijoDrumFrameDL, gTijoRideCymbalDL, gTijoCrashCymbalDL, gTijoHiHatDL, gTijoDrum1DL,
+        gTijoDrum2DL,     gTijoDrum3DL,      gTijoDrum4DL,       gTijoDrum5DL, gTijoBassDrumDL,
+    };
+    f32 instrumentPosXs[] = { 0.0f, -2690.0f, 2310.0f, 3888.0f, -4160.0f, -2200.0f, -463.0f, 1397.0f, 3413.0f, 389.0f };
+    f32 instrumentPosYs[] = { 0.0f, 6335.0f, 6703.0f, 5735.0f, 3098.0f, 3349.0f, 3748.0f, 3718.0f, 2980.0f, 1530.0f };
+    f32 instrumentPosZs[] = { 0.0f, 4350.0f, 3200.0f, 1555.0f, 2874.0f, 3901.0f, 4722.0f, 4344.0f, 3200.0f, 3373.0f };
+    f32 scale;
+
+    OPEN_DISPS(play->state.gfxCtx);
+
+    for (i = 0; i < ARRAY_COUNT(sTijoDrumsDLs); i++) {
+        Matrix_Push();
+        Matrix_Translate(instrumentPosXs[i], instrumentPosYs[i], instrumentPosZs[i], MTXMODE_APPLY);
+
+        switch (i) {
+            case ENZOD_INSTRUMENT_CYMBAL_1:
+            case ENZOD_INSTRUMENT_CYMBAL_2:
+            case ENZOD_INSTRUMENT_CYMBAL_3:
+                Matrix_RotateXS(this->cymbalRots[i - 1], MTXMODE_APPLY);
+                break;
+
+            case ENZOD_INSTRUMENT_DRUM_1:
+            case ENZOD_INSTRUMENT_DRUM_2:
+            case ENZOD_INSTRUMENT_DRUM_3:
+            case ENZOD_INSTRUMENT_DRUM_4:
+            case ENZOD_INSTRUMENT_DRUM_5:
+            case ENZOD_INSTRUMENT_BASS_DRUM:
+                scale = this->drumScales[9 - i] + 1.0f;
+                Matrix_Scale(scale, scale, scale, MTXMODE_APPLY);
+                break;
+
+            default:
+                break;
+        }
+
+        gSPMatrix(POLY_OPA_DISP++, Matrix_NewMtx(play->state.gfxCtx), G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+        gSPDisplayList(POLY_OPA_DISP++, sTijoDrumsDLs[i]);
+        Matrix_Pop();
+    }
+
+    CLOSE_DISPS(play->state.gfxCtx);
+}
+
+void EnZod_Draw(Actor* thisx, PlayState* play) {
+    static TexturePtr sTijoEyesTextures[] = { &gTijoEyesOpen, &gTijoEyesHalfOpen, &gTijoEyesClosed };
+    EnZod* this = THIS;
+    Gfx* gfxP;
```

Comment:
```suggestion
    Gfx* gfx;
```
gfxP is generally reserved for `Gfx**`